cmake/getversion.cmake

```diff
@@ -201,7 +201,7 @@ if(PROJECT_VERSION_LABEL)
 	set(PROJECT_VERSION "${PROJECT_VERSION}-${PROJECT_VERSION_LABEL}")
 endif()
 
-set(PROJECT_VERSION_FRIENDLY "${PROJECT_VERSION} (${GIT_COMMIT_DATE_FRIENDLY})")
+set(PROJECT_VERSION_FRIENDLY "${PROJECT_VERSION}<br>(${GIT_COMMIT_DATE_FRIENDLY})")
```

Comment:
I guess that's also the reason for this faulty display:
![untitled](https://cloud.githubusercontent.com/assets/9874850/25555771/54c23040-2cef-11e7-9880-29b9005a680d.png)